internal/controller/biossettingsset_controller.go

```diff
@@ -141,16 +140,18 @@ func (r *BIOSSettingsSetReconciler) handleBiosSettings(
 	}
 
 	if err := r.createMissingBIOSSettings(ctx, log, serverList, ownedBiosSettings, biosSettingsSet); err != nil {
```

Comment:
can't we use the same func `patchOrCreateBIOSSettingsfromTemplate`, since it also creates BIOSSettings as well..